docs/PROI/report.tex

```diff
@@ -11,11 +13,58 @@
 \tableofcontents
 
 \section{Einleitung}
-\subsection{Projekt}
 \subsection{Unternehmen}
+\colorbox{red}{was macht macio aus}
+\subsection{Projektidee}
+Im Rahmen des Projekt Informatik möchte Macio ihr Portfolio im IoT-Bereich erweitern sowie ihren Empfangsraum im Standort Kiel verschönern.
+Hierfür soll eine smarte Spielzeug-Box gebaut werden.
+Smarte Spielzeuge gibt es im kommerziellen Bereich viele, daher soll dieses Projekt eine Open-Source-Alternative schaffen.\\
+Genauer handelt es sich um eine Musik-Box, die NFC-Chips lesen und Spotify Connect unterstützen soll.
+Auf die Box können dann Spielzeuge (z.B. in Form von kleinen Figuren) mit integrierten NFC-Chips gestellt werden, um spezifische Musik abspielen zu lassen.
+Die Musik wird von Spotify-Connect über eine bereits bestehende Musik-Anlage abgespielt.
+Falls es im Rahmen des Projektes möglich ist, sollen die Nutzer in der Lage sein, zwischen verschiedenen Musikanbietern zu wechseln.
+NFC-Chips und die zugehörige Musik sollen über ein Web-basierte Benutzeroberfläche konfiguriert werden können.
+Diese Nutzeroberfläche soll von der Box ausgeliefert und primär für Smartphone-Bedienung gestaltet werden.
```

Comment:
Entweder Benutzeroberfläche oder Nutzeroberfläche, aber nicht zwei Begriffe in zwei direkt aufeinander folgenden Sätzen.

Reply:
```suggestion
Diese Benutzeroberfläche soll von der Box ausgeliefert und primär für Smartphone-Bedienung gestaltet werden.
```